types/notification.ts

```diff
@@ -72,28 +70,38 @@ export interface NotificationDisableOption {
 }
 
 export enum NotifiedActionType {
-  CustomDaoCreation = 'CustomDaoCreation',
-  ClubDaoCreation = 'ClubDaoCreation',
-  FoundationDaoCreation = 'FoundationDaoCreation',
-  CorporationDaoCreation = 'CorporationDaoCreation',
-  CooperativeDaoCreation = 'CooperativeDaoCreation',
-
-  TransferProposalCreation = 'TransferProposalCreation',
-  BountyProposalCreation = 'BountyProposalCreation',
-  BountyDoneProposalCreation = 'BountyDoneProposalCreation',
-  PollProposalCreation = 'PollProposalCreation',
-  DaoNameUpdated = 'DaoNameUpdated',
-  DaoPurposeUpdated = 'DaoPurposeUpdated',
-  DaoLegalUpdated = 'DaoLegalUpdated',
-  DaoLinksUpdated = 'DaoLinksUpdated',
-  DaoFlagUpdated = 'DaoFlagUpdated',
-  DaoDeadlinesUpdated = 'DaoDeadlinesUpdated',
-  DaoRulesUpdated = 'DaoRulesUpdated',
-  DaoGroupAdded = 'DaoGroupAdded',
-  DaoGroupUpdated = 'DaoGroupUpdated',
-  DaoGroupRemoved = 'DaoGroupRemoved',
-  DaoMembersAdded = 'DaoMembersAdded',
-  DaoMemberRemoved = 'DaoMemberRemoved',
+  CustomDao = 'CustomDao',
+  ClubDao = 'ClubDao',
+  FoundationDao = 'FoundationDao',
+  CorporationDao = 'CorporationDao',
+  CooperativeDao = 'CooperativeDao',
+
+  AddMemberToRole = 'AddMemberToRole',
+  RemoveMemberFromRole = 'RemoveMemberFromRole',
+  FunctionCall = 'FunctionCall',
+  Transfer = 'Transfer',
+  ChangePolicy = 'ChangePolicy',
+  ChangeConfig = 'ChangeConfig',
+  AddBounty = 'AddBounty',
+  BountyDone = 'BountyDone',
+  Vote = 'Vote',
+
+  // TransferProposalCreation = 'TransferProposalCreation',
```

Comment:
Should we keep this comments?

Reply:
These are the old types, I kept them just for reference. Ill remove them once we agreed about all possible notifications